.gitattributes

```
@@ -1,3 +1,6 @@
 * text=auto eol=lf
 *.{cmd,[cC][mM][dD]} text eol=crlf
 *.{bat,[bB][aA][tT]} text eol=crlf
+
+.automation/generated/flavor-stats.json linguist-generated=true
+.automation/generated/linter-helps.json linguist-generated=true
```

If we want to just add it, once discussed
```suggestion
.automation/generated/linter-helps.json linguist-generated=true
docs/descriptors/python_black.md linguist-generated=true
docs/descriptors/repository_checkov.md linguist-generated=true
docs/descriptors/terraform_checkov.md linguist-generated=true
```